iRODS.cyberduckprofile

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2002-2017 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation; either version 2 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>irods</string>
+        <key>Vendor</key>
+        <string>iRODS Consortium</string>
+        <key>Description</key>
+        <string>iRODS (Integrated Rule-Oriented Data System)</string>
+
+        <key>Hostname Configurable</key>
+        <true/>
+        <key>Port Configurable</key>
+        <true/>
+
+        <key>Default Port</key>
+        <string>1247</string>
+
+        <!-- The zone of the iRODS server to connect to. -->
+        <!-- For backward compatibility, a root resource can be specified by appending ":<resouce>" to the zone. -->
+        <!-- The specified resource is only used when uploading files. -->
+        <!-- Prefer setting "Destination Resource" under the "Properties" section. -->
+        <key>Region</key>
+        <string>tempZone</string>
```

Comment:
Can this be kept empty to connect to a default zone or is this zone always available?

Reply:
Not at this time. All iRODS clients are required to manually define the zone in order to connect.